pkg/rehearse/jobs.go

```diff
@@ -111,8 +111,22 @@ func makeRehearsalPresubmit(source *prowconfig.Presubmit, repo string, prNumber
 	rehearsal.Context = fmt.Sprintf("ci/rehearse/%s/%s/%s", repo, branch, shortName)
 	rehearsal.RerunCommand = defaultRehearsalRerunCommand
 
-	gitrefArg := fmt.Sprintf("--git-ref=%s@%s", repo, branch)
-	rehearsal.Spec.Containers[0].Args = append(source.Spec.Containers[0].Args, gitrefArg)
+	orgRepo := strings.Split(repo, "/")
+
+	// We will never hit different length because it will fail while we are loading the prowconfig.
+	// This check is to satisfy the tests and make sure that we won't hit any panic in case of any
+	// bugs while loading the prow config.
+	// It is go idiom that the string{"org","repo"} has length equals to '2'.
+	if len(orgRepo) > 1 {
+		rehearsal.ExtraRefs = []pjapi.Refs{
+			{
+				Org:     orgRepo[0],
+				Repo:    orgRepo[1],
+				BaseRef: branch,
+				WorkDir: true,
```

Comment:
What happens if we attempt to rehearse something that already has extra refs? Do we just overwrite them?

Reply:
I didn't think of that. Nice catch.. I will address this. Thanks.

Reply:
addressed